package.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
What are the significance of adding npm related stuff in android project?
@W3-Anjan

Reply:
For automatic semantic versioning and changelog creation I have tried with several ways from internet. There some tutorial of javascript also but [ this reference](https://www.mokkapps.de/blog/how-to-automatically-generate-a-helpful-changelog-from-your-git-commit-messages/) works well. And already with this tutorial automated change log, release tag with semantic versioning is working @sabbir360 vai